test/karma/dataBehaviorSpec.js

```diff
@@ -131,6 +131,54 @@ describe('A Torso Data Behavior', function() {
       }
     });
 
+    it('defaults skipInitialLoad to false', function() {
+      var defaultBehaviorConfigurationSkipInitialLoadUndefined = getBasicBehaviorConfiguration();
+      delete defaultBehaviorConfigurationSkipInitialLoadUndefined.skipInitialLoad;
+      var ViewWithDataBehaviorSkipInitialLoadUndefined = TorsoView.extend({
+        behaviors: {
+          dataBehavior: defaultBehaviorConfigurationSkipInitialLoadUndefined
+        }
+      });
+      var viewWithDataBehaviorSkipInitialLoadUndefined = new ViewWithDataBehaviorSkipInitialLoadUndefined();
+      expect(viewWithDataBehaviorSkipInitialLoadUndefined.getBehavior('dataBehavior').skipInitialLoad).toBe(false);
+    });
+
+    it('can set skipInitialLoad to false', function() {
+      var defaultBehaviorConfigurationSkipInitialLoadFalse = getBasicBehaviorConfiguration();
+      defaultBehaviorConfigurationSkipInitialLoadFalse.skipInitialLoad = false;
+      var ViewWithDataBehaviorSkipInitialLoadFalse = TorsoView.extend({
+        behaviors: {
+          dataBehavior: defaultBehaviorConfigurationSkipInitialLoadFalse
+        }
+      });
+      var viewWithDataBehaviorSkipInitialLoadFalse = new ViewWithDataBehaviorSkipInitialLoadFalse();
+      expect(viewWithDataBehaviorSkipInitialLoadFalse.getBehavior('dataBehavior').skipInitialLoad).toBe(false);
+    });
+
+    it('can set skipInitialLoad to true', function() {
+      var defaultBehaviorConfigurationSkipInitialLoadTrue = getBasicBehaviorConfiguration();
+      defaultBehaviorConfigurationSkipInitialLoadTrue.skipInitialLoad = true;
+      var ViewWithDataBehaviorSkipInitialLoadTrue = TorsoView.extend({
+        behaviors: {
+          dataBehavior: defaultBehaviorConfigurationSkipInitialLoadTrue
+        }
+      });
+      var viewWithDataBehaviorSkipInitialLoadTrue = new ViewWithDataBehaviorSkipInitialLoadTrue();
+      expect(viewWithDataBehaviorSkipInitialLoadTrue.getBehavior('dataBehavior').skipInitialLoad).toBe(true);
+    });
+
+    it('defaults returnSingleResult to false', function() {
```

Comment:
Is this a duplicate? Looks the same as the one below?

Reply:
fixed